README.md

```diff
@@ -56,6 +56,16 @@ $ cat /usr/share/ramalama/shortnames.conf
 
 ## Install
 
+## Install via PyPy
+
+Ramalama is available via PyPy [https://pypi.org/project/ramalama](https://pypi.org/project/ramalama)
```

Comment:
It's PyPI, not PyPy (the latter is a different Python implementation).

Reply:
Thanks

Reply:
PS, the PR title also says PyPy.